src/nvim/eval/typval.h

```diff
@@ -469,7 +84,7 @@ static inline void list_log(const list_T *const l, const listitem_T *const li1,
 # define list_free_log()
 #endif
 
-// In a hashtab item "hi_key" points to "di_key" in a dictitem.
+/// Ref// In a hashtab item "hi_key" points to "di_key" in a dictitem.
```

Comment:
?

Reply:
Oof it seems like an accidental copy/paste. Fixed.